packages/components/psammead-headings/README.md

```diff
@@ -51,7 +51,9 @@ This usage will allow for the page anchor: `www.bbc.com/news/articles/articleID#
 
 ### When to use this component
 
-These components can be used at any point on the page, however the `Headline` and `PageHeading` are designed to be used once at the top of the page. The `SubHeading` takes an optional `id` value and passes it to the `h2` which can be used as an anchor when referencing content.
+These components can be used at any point on the page, however the `Headline` and `PageHeading` are designed to be used once at the top of the page.
+
+The `SubHeading` and `PageHeading` can take an optional `id` attribute to be passed to the `h2` which can be used as an anchor when referencing content.
```

Comment:
h2?

Reply:
I'll correct this since the `SubHeading` component uses a `h2` but not the `PageHeading`